lib/tropic/tropic_paragraph.flow

```diff
@@ -962,6 +986,42 @@ commonWordConstructor(
 	);
 }
 
+calculateWordTextAndMetrics(
```

Comment:
It's not so obvious that calculateWordTextAndMetrics both updates inspector and returns collected data. I'd add a comment to prevent confusion.